procdiscovery/pkg/inspectors/java/java.go

```diff
@@ -11,10 +11,13 @@ type JavaInspector struct{}
 
 const processName = "java"
 
-func (j *JavaInspector) Inspect(p *process.Details) (common.ProgrammingLanguage, bool) {
+func (j *JavaInspector) Inspect(p *process.Details) (common.ProgramLanguageDetails, bool) {
+	var programLanguageDetails common.ProgramLanguageDetails
+
 	if strings.Contains(p.ExeName, processName) || strings.Contains(p.CmdLine, processName) {
-		return common.JavaProgrammingLanguage, true
+		programLanguageDetails.Language = common.JavaProgrammingLanguage
```

Comment:
can we detect the java runtime version as well?

Reply:
Eden said to start with Python and Node, later we will add more languages.
I can look if it is super straight forward I'll add.

Reply:
Yes we can :)